src/borg/testsuite/archiver/argparsing.py

```diff
@@ -11,6 +12,23 @@ def test_bad_filters(self):
         self.cmd(f"--repo={self.repository_location}", "create", "test", "input")
         self.cmd(f"--repo={self.repository_location}", "delete", "--first", "1", "--last", "1", fork=True, exit_code=2)
 
+    def test_highlander(self):
+        self.cmd(f"--repo={self.repository_location}", "rcreate", RK_ENCRYPTION)
+        self.cmd(f"--repo={self.repository_location}", "create", "--comment", "comment 1", "test-1", src_dir)
```

Comment:
maybe use something else than "src_dir" - that is a rather expensive thing, archives the complete source directory of borg.

Reply:
I think its only the testsuite dir?

Reply:
right. but still too much stuff.